packages/former/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "former-ui",
   "type": "module",
-  "version": "0.0.11-next.55",
+  "version": "0.0.0",
```

Comment:
@lukashass Why do you have removed the version number from here? I would like to keep it as it seems to be pretty normal when e.g. looking at the vue or vitest repos.

Reply:
@mariusheine But what do you set here? The version is taken from the git tag now and having it twice doesn't make sense.

Reply:
@mariusheine Of course it needs to be set in the published package. That is what `pnpm version` in release.yaml does :wink: